common/src/event.rs

```diff
@@ -217,6 +216,12 @@ pub type UserMessageReadReason = Reason<UserMessageReadRuntimeReason, UserMessag
 #[codec(crate = codec)]
 #[scale_info(crate = scale_info)]
 pub enum ProgramChangeKind<BlockNumber> {
+    /// Occurs when new program has been added to storage.
+    ///
+    /// Expiration block number presents block number when this program become
+    /// paused due to losing ability to pay rent for holding.
+    Added { expiration: BlockNumber },
```

Comment:
To be considered to have such event

Reply:
discussed: we need such event indeed though it will be renamed to `ExpirationChanged`. One more event for terminated programs should be also introduced.